test_util/dcos_api_session.py

```diff
@@ -88,7 +88,7 @@ def api_request(self, method, path_extension, *, scheme=None, host=None, query=N
                                    query=query, fragment=fragment, port=port, **kwargs)
 
 
-class DcosApiSession(ARNodeApiClientMixin, ApiClientSession):
+class DcosApiSession(ARNodeApiClientMixin, RetryCommonHttpErrorsMixin, ApiClientSession):
```

Comment:
Sounds good, just wanted to double check since I don't know how the inheritance in python works (haven't learned it yet, since I've trained myself away from most OO inheritance the past couple years.)